content/en/docs/tasks/job/indexed-parallel-processing-static.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Indexed Job for Parallel Processing with Static Work Assignment
+content_type: task
+min-kubernetes-server-version: v1.21
+weight: 30
+---
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+<!-- overview -->
+
+{{< note >}}
+To be able to create Indexed Jobs, make sure to enable the `IndexedJob`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](docs/reference/command-line-tools-reference/kube-apiserver/)
+and the [controller manager](/docs/reference/command-line-tools-reference/kube-controller-manager/).
+{{< /note >}}
+
+In this example, we will run a Kubernetes Job with multiple parallel worker processes.
+Each worker Pod gets assigned a unit of work statically, which is determined by
+the pod index available in the {{< glossary_tooltip text="annotation" term_id="annotation" >}}
+`batch.alpha.kubernetes.io/job-completion-index`, also accessible to containers
+in the `JOB_COMPLETION_INDEX` environment variable.
+
+Here is an overview of the steps in this example:
+
+1. **Create an image that can read the pod index**
+2. **Start an Indexed Job**. The downward API allows you to pass the annotation
+   as an environment variable or file to the container.
+
+<!-- steps -->
+
+## Create an Image
```

Comment:
nit:
```suggestion
## Create a container image
```

Lowercasing “Image” because people don't `kubectl create image` (and headings in documentation are sentence case).